actors/miner/src/lib.rs

```diff
@@ -4559,7 +4558,7 @@ fn termination_penalty(
             &sector_power,
             reward_estimate,
             &sector.replaced_day_reward,
-            sector.replaced_sector_age,
+            sector.power_base_epoch - sector.activation,
```

Comment:
Should 4555 be `current_epoch - sector.power_base_epoch` (and can we test the difference)?

Reply:
In the original CC path, the terminated sector's activation is the _replacement epoch_, not the original sector's activation, because it was an actually distinct sector.`total_lifetime` is only since replacement. So from your example, the penalty is:
- 90*R if terminated at epoch 120 (or later)
- 60*R + 30*D if terminated at epoch 90
- 15*R + 30*D if terminated at epoch 45

After this PR, calculating the sector's age as `current_epoch - sector.power_base_epoch` will maintain this intent, because we're moving `sector.activation` back to the true activation epoch. The previous code's value is correct only because `sector.activation` isn't, but it must be changed if we change (fix) the recorded activation epoch.